Web-Scraping/Medium-Articles-Details-Scrapping/run.py

```diff
@@ -0,0 +1,10 @@
+from app import Scrap
+
+
+a = Scrap(urls_dict={"Towards Data Science": "https://towardsdatascience.com",
```

Comment:
Use Dynamic Inputs instead of hardcoded ones

Reply:
+1 from my side!